pythran/tests/test_ndarray.py

```diff
@@ -13,15 +13,25 @@
     has_float128 = False
 
 
+huge = numpy.iinfo(numpy.intp).max // numpy.intp().itemsize
+
+def raisesMemoryError():
+    try:
+        numpy.ones(huge)
```

Comment:
If I understand this correctly, this might raise ValueError instead. Should that exception be checked and return False?

Reply:
Agreed & Fixed

Reply:
(Or so I hope)